src/init.cpp

```diff
@@ -1176,6 +1176,10 @@ bool AppInitParameterInteraction(const ArgsManager& args)
 
     nMaxTipAge = args.GetArg("-maxtipage", DEFAULT_MAX_TIP_AGE);
 
+    if (args.IsArgSet("-proxy") && args.GetArg("-proxy", "").empty()) {
+        return InitError(Untranslated("No proxy server specified. Use -proxy=<ip> or -proxy=<ip:port>."));
```

Comment:
3bca86ae6b7cc5b2f5bb9ee4567052b34f613ba1
```suggestion
        return InitError(_("No proxy server specified. Use -proxy=<ip> or -proxy=<ip:port>."));
```

as the message is presented to a user in GUI:

![Screenshot from 2020-09-23 17-22-22](https://user-images.githubusercontent.com/32963518/94025911-b8d98500-fdc1-11ea-9305-810df2ca63a9.png)

Reply:
Done!